t/sharness.d/flux-accounting.sh

```diff
@@ -0,0 +1,5 @@
+FLUX_EXEC_PATH_PREPEND=${SHARNESS_TEST_SRCDIR}/../src/cmd:${FLUX_EXEC_PATH}
```

Comment:
No need to add `FLUX_EXEC_PATH` to `FLUX_EXEC_PATH_PREPEND` here.

Reply:
Oops, sorry about that - I squashed and pushed a fix to remove `FLUX_EXEC_PATH` from `FLUX_EXEC_PATH_PREPEND`.